src/Appwrite/Vcs/Comment.php

```diff
@@ -149,10 +151,11 @@ public function generateComment(): string
                 $text .= "| :- | :-  | :-  | :- |\n";
 
                 foreach ($project['function'] as $functionId => $function) {
-                    $extension = $function['status'] === 'building' ? 'gif' : 'png';
+                    $imageStatus = in_array($function['status'], ['processing', 'building']) ? 'building' : $function['status'];
```

Comment:
![CleanShot 2025-06-02 at 10 38 30@2x](https://github.com/user-attachments/assets/0ff22521-0f7a-4cc4-976b-09c026913214)

Seems like we have processing state, it's just called waiting. Let's map that specific word, and we should be good

Reply:
`waiting` and `processing` are different states. `processing` and `building` should use the same gif.